kapitan/utils.py

```diff
@@ -530,6 +531,11 @@ def make_request(source):
 
 def unpack_downloaded_file(file_path, output_path, content_type):
     """unpacks files of various MIME type and stores it to the output_path"""
+
+    if (content_type == None or content_type == "application/octet-stream"):
+      if (re.search(r"^Zip archive data.*", magic.from_file(file_path))):
+        content_type = 'application/zip'
```

Comment:
@metacoma please can you update spacing to 4?

Reply:
@ramaro hey, fixed and excuse me about this, my vim is not configured properly ATM :(